cdk/domino_cdk/config/iam.py

```diff
@@ -0,0 +1,412 @@
+            "eks:DescribeNodegroup",
+            "eks:TagResource",
+            "eks:UntagResource",
+            "elasticfilesystem:Backup",
+            "elasticfilesystem:CreateMountTarget",
+            "elasticfilesystem:DeleteMountTarget",
+            "elasticfilesystem:DescribeFileSystemPolicy",
+            "elasticfilesystem:DescribeMountTargets",
+            "elasticfilesystem:ListTagsForResource",
+            "elasticfilesystem:TagResource",
+            "elasticfilesystem:UntagResource",
+            "iam:GetRole",
+        ],
+        **do_cf(),
+        "Resource": "*",
+    }
+    if not manual:
+        general["Action"].extend(
+            [
+                "backup-storage:MountCapsule",
+                "backup:*BackupPlan",
+                "backup:*BackupSelection",
+                "ec2:AssociateRouteTable",
+                "ec2:AssociateVpcCidrBlock",
+                "ec2:AttachInternetGateway",
+                "ec2:AuthorizeSecurityGroupEgress",
+                "ec2:AuthorizeSecurityGroupIngress",
+                "ec2:CreateInternetGateway",
+                "ec2:CreateNatGateway",
+                "ec2:CreateRoute",
+                "ec2:CreateRouteTable",
+                "ec2:CreateVpc",
+                "ec2:CreateVpcEndpoint",
+                "ec2:DeleteInternetGateway",
+                "ec2:DeleteNatGateway",
+                "ec2:DeleteRoute",
+                "ec2:DeleteRouteTable",
+                "ec2:DeleteVpc",
+                "ec2:DeleteVpcEndpoints",
+                "ec2:DescribeInternetGateways",
+                "ec2:DescribeNatGateways",
+                "ec2:DescribeRouteTables",
+                "ec2:DescribeVpcEndpoints",
+                "ec2:DescribeVpcs",
+                "ec2:DetachInternetGateway",
+                "ec2:DisAssociateRouteTable",
+                "ec2:DisassociateVpcCidrBlock",
+                "ec2:ModifyVpcAttribute",
+                "ec2:RevokeSecurityGroupEgress",
+                "ec2:RevokeSecurityGroupIngress",
+                "elasticfilesystem:CreateAccessPoint",
+                "elasticfilesystem:CreateFileSystem",
+                "elasticfilesystem:DeleteAccessPoint",
+                "elasticfilesystem:DeleteFileSystem",
+                "elasticfilesystem:DescribeAccessPoints",
+                "elasticfilesystem:DescribeFileSystems",
+                "iam:ListAttachedRolePolicies",
+            ]
+        )
+    else:
+        general["Action"].extend(
+            [
+                "ec2:*InternetGateway*",
+                "ec2:*NatGateway*",
+                "ec2:*Route*",
+                "ec2:*SecurityGroupEgress",
+                "ec2:*SecurityGroupIngress",
+                "ec2:*Vpc*",
+                "s3:GetBucketLocation",
+                "s3:GetObject",
+                "s3:ListBucket",
+            ]
+        )
+        general["Resource"] = ["*"]
+    general["Action"] = sorted(general["Action"])
+
+    base_iam = {
```

Comment:
Where is the genius approach to pack permissions into 2 policies?

Reply:
That was going to be future work, and not part of this PR, some PR that merges the policies and then splits them. This was step 1. However, Steve already hit the "long name" test you'd mention, so I will probably make the two files/two policies before I merge this (but the merging of the policies will still be followup work).

Reply:
I implemented the split part.